docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1583,8 +1583,20 @@ You can use the [`collapse-rename-collapsedheight` codemod](https://github.com/m
   +<Tabs scrollButtons={false} />
   ```
 
+### Tab
+
```

Comment:
The headers of the migration guide match more or less the label we have. The `Tab` component is part of the Tabs. So I think that we should do:
```suggestion
```

Reply:
The Tab is a separate component so it should have a separate heading. The uppercasing clearly indicates a component not some internal notion of "component groups".

Reply:
The `Tab` and `Tabs` React components are part of the same [tabs](https://www.w3.org/TR/wai-aria-practices/#tabpanel) component. Now, it depends on how we see these headers, are they React components or UI components? So far, the structure of the guide depends on who wrote it. On the order I find the information reading top-down

- @mbrookes -> UI component, e.g. AlertTitle
- Olivier -> UI component, e.g. DialogTitle
- Petyo -> React component, e.g. FomControl
- Jun -> React component, e.g. MenuItem
- @m4theushw -> React component, e.g. MenuItem, UI component e.g. TablePagination

Happy as long as we are consistent with the approach.

Reply:
Considering the context in which developers will consume this content, I would +1 for the React component approach. So, not doing the first change I have proposed.